vxquery-core/src/main/java/org/apache/vxquery/datamodel/values/XDMConstants.java

```diff
@@ -32,6 +32,8 @@
 
     private static final byte[] EMPTY_SEQUENCE;
 
+    private static final byte[] EMPTY_OBJECT;
```

Comment:
Do you think we need this constant? How often will you have an empty object as a result?